tests/phpunit/integration/ContextTest.php

```diff
@@ -71,7 +71,7 @@ public function test_filter_input() {
 		$this->assertTrue( $context->input()->filter( INPUT_GET, 'foo', FILTER_VALIDATE_BOOLEAN ) );
 
 		$_GET['dirty'] = '<script>dirt</script>';
-		$this->assertEquals( 'dirt', $context->input()->filter( INPUT_GET, 'dirty', FILTER_SANITIZE_STRING ) );
+		$this->assertEquals( '&lt;script&gt;dirt&lt;/script&gt;', htmlspecialchars( $context->input()->filter( INPUT_GET, 'dirty' ) ) );
```

Comment:
This test doesn't seem so useful now as it's effectively just testing the standard `htmlspecialchars` function. I'd suggest it should be removed.
```suggestion
```